event-sourcing/event-sourcing-core/pom.xml

```diff
@@ -59,6 +59,10 @@
             <artifactId>equalsverifier</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+        </dependency>
```

Comment:
why do you need it?

Reply:
It's needed here:
https://github.com/linagora/james-project/blob/1968a528f61ee0c16515177b99919825ab3e0597/event-sourcing/event-sourcing-core/src/main/java/org/apache/james/eventsourcing/EventBus.java#L27

Guavate is bringing Guava & Commons-lang3 so I had to add them when needed & not declared.